packages/css/src/grid/grid.scss

```diff
@@ -20,3 +19,80 @@
     grid-template-columns: repeat(var(--amsterdam-grid-wide-column-count), 1fr);
   }
 }
+
+.amsterdam-grid--density-high {
+  gap: var(--amsterdam-grid-density-high-gap);
+  padding-inline: var(--amsterdam-grid-density-high-padding-inline);
+}
+
+.amsterdam-grid--density-low--margin-bottom--small {
+  padding-block-end: calc(var(--amsterdam-grid-density-low-gap) / 2);
```

Comment:
We're setting the `padding`, but the class is called `*-margin-*`, that's a bit odd